shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -292,6 +298,10 @@ public long getIDTokenExpiry() {
         return Long.parseLong(System.getenv().getOrDefault("ID_TOKEN_EXPIRY", "120"));
     }
 
+    public Optional<String> getNotifyApiUrl() {
+        return Optional.ofNullable(System.getenv("NOTIFY_URL"));
+    }
+
```

Comment:
Why did this change?

Reply:
it was deleted by accident in a previous commit it was just added back